README.md

```diff
@@ -121,5 +121,10 @@ Pull requests are welcome. For major changes, please open an issue first to disc
 
 Please make sure to update tests as appropriate.
 
+## Contributors
+There are two contributors to this project.
```

Comment:
Incorrect, the ask was to link to the GitHub profiles of each contributor. And in the process, pick up some rudimentary markdown skills